site/plugins.json

```diff
@@ -621,5 +621,13 @@
     "package": "netlify-plugin-build-logger",
     "repo": "https://github.com/amar-p6/netlify-build-logger-plugin.git",
     "version": "1.0.3"
+  },
+  {
+    "author": "aaronbassett",
+    "description": "The New Relic Netlify plugin automatically notifies New Relic of Netlify build events and installs the New Relic browser agent.",
+    "name": "New Relic",
+    "package": "@newrelic/newrelic-netlify-plugin",
```

Comment:
Hi there!
We usually follow a naming convention documented [here](https://docs.netlify.com/configure-builds/build-plugins/share-plugins/#publish-to-npm) which would lead to a package name more along the lines of either `@newrelic/netlify-plugin` or `@newrelic/netlify-plugin-some-suffix`.
Would this be possible to rename the npm package name? If that's too much of a problem though, the current name works too :+1: